reconcile/cli.py

```diff
@@ -66,7 +67,26 @@ def before_breadcrumb(crumb, hint):
 
 # Enable Sentry
 if os.getenv("SENTRY_DSN"):
-    sentry_sdk.init(os.environ["SENTRY_DSN"], before_breadcrumb=before_breadcrumb)
+    match os.environ.get("SENTRY_EVENT_LEVEL", "CRITICAL").upper():
+        case "CRITICAL":
+            sentry_event_level = logging.CRITICAL
+        case "ERROR":
+            sentry_event_level = logging.ERROR
+        case _:
+            raise ValueError(
+                "Invalid value for SENTRY_EVENT_LEVEL. Must be CRITICAL or ERROR."
+            )
+
+    sentry_sdk.init(
+        os.environ["SENTRY_DSN"],
+        before_breadcrumb=before_breadcrumb,
+        integrations=[
+            LoggingIntegration(
+                # send critical errors as events
```

Comment:
note the comment says `critical` but can be `error` depends on env.

Reply:
Removed the misleading comment